Gemfile.lock

```diff
@@ -580,9 +582,7 @@ DEPENDENCIES
   turbolinks
   uglifier (>= 1.3.0)
   unobtrusive_flash (>= 3)
-  web-console (~> 2.0)
-  webmock
-  whenever
```

Comment:
Why are we removing these 3 lines? Did you do that manually?
I think @ChrisBr 's comment might have been about the lines below this (lines 585-588), as the comment was on the empty line, not on the ones you removed. 
In any case, you cannot remove gems from the Gemfile.lock while you still have them in your Gemfile. That's what's causing travis to fail.